src/main/java/seedu/address/model/person/patient/Patient.java

```diff
@@ -76,10 +76,10 @@ public boolean equals(Object other) {
             return false;
         }
 
-        Patient otherPerson = (Patient) other;
-        return name.equals(otherPerson.name)
-                && phone.equals(otherPerson.phone)
-                && nric.equals(otherPerson.nric);
+        Patient otherPatient = (Patient) other;
+        return name.equals(otherPatient.name)
+                && phone.equals(otherPatient.phone)
```

Comment:
is there a need to compare name and phone? would comparing nric be enough to identify each patient/doctor since nric should be unique?

Reply:
we have 2 methods, equals and isSame. Equal will check all attribute while isSame only checks NRIC.